java/jdbc/src/main/java/io/vitess/jdbc/ConnectionProperties.java

```diff
@@ -204,6 +204,11 @@
         "The password protecting the keystore file (if a password is set)",
         null,
         null);
+    private StringConnectionProperty keyStoreType = new StringConnectionProperty(
+        Constants.Property.KEYSTORE_TYPE,
+        "The type of store used for the key (default is JKS)",
+        null,
+        null);
```

Comment:
Give default keyStoreType in connection property

StringConnectionProperty(String name, String description, String defaultValue, String[] allowableValuesToSet)